.golangci.yml

```diff
@@ -0,0 +1,241 @@
+run:
+  timeout: 10m
+  allow-parallel-runners: true
+
+  # Maximum issues count per one linter. Set to 0 to disable. Default is 50.
+  max-issues-per-linter: 0
+
+  # Maximum count of issues with the same text. Set to 0 to disable. Default is 3.
+  max-same-issues: 0
+linters:
+  disable-all: true
+  enable:
+    - asasalint
+    - asciicheck
+    - bidichk
+    - bodyclose
+    - contextcheck
+    - decorder
+    - dogsled
+    - dupl
+    - durationcheck
+    - errcheck
+    - errchkjson
+    - errname
+    - execinquery
+    - ginkgolinter
+    - gocheckcompilerdirectives
+    - goconst
+    - gocritic
+    - gocyclo
+    - godox
+    - gofmt
```

Comment:
golangci supports gofmt linter, so we can remove the old one

Reply:
yup, let's do on a followup :)

Reply:
Created a item to track: https://github.com/kubernetes/ingress-nginx/issues/10195